### PR TITLE
feat(registrar): display a comprehensive error when trying to resend …

### PR DESCRIPTION
**Pull request by xflord**
…email verification for already approved application

### CODE REVIEW

**Comment by xflord** [comment]
Would be good if someone else tested this, I didn't manage to replicate this error. (Even after approving the application, when I hit resend the "success window" popped up)

**Comment by Johaney-s** [comment]
Tested it and would change "application has been approved" to something more generic, because this shows even when the application was rejected. Also the spinner keeps spinning next to the "Re-send mail verification message" button - see picture
image

File: perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/steps/SummaryStep.java
Comment (Johaney-s) [inline perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/steps/SummaryStep.java:663]: looks good now, one more thing is the last dialog about "waiting for admin to approve the application", I think that can be removed so it is not confusing: 
image

**Review by Johaney-s — APPROVED** [review]

**Review by Kuliak — APPROVED** [review]

**Pull request merged**
